client/pages/EventFeed.tsx

```diff
@@ -1,8 +1,7 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import Comments from '../components/Comments';
-
-import { styled, Button, Card, CardHeader, CardMedia, CardContent, CardActions, Collapse, Avatar, Typography, IconButton } from '@mui/material';
+import { OutlinedInput, Fab, styled, Button, Card, CardHeader, CardMedia, CardContent, CardActions, Collapse, Avatar, Typography, IconButton } from '@mui/material';

```

Comment:
do you want to use named imports here? you're importing the entire @mui/material library. check out the docs on [minimizing bundle size](https://mui.com/material-ui/guides/minimizing-bundle-size/)